.vscode/launch.json

```diff
@@ -0,0 +1,7 @@
+{
+	// Use IntelliSense to learn about possible attributes.
+	// Hover to view descriptions of existing attributes.
+	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+	"version": "0.2.0",
+	"configurations": []
+}
```

Comment:
Do we need this file on VCS?

Reply:
no, we don't need them so I remove them.